pkg/apis/core/validation/validation.go

```diff
@@ -3776,32 +3778,43 @@ const (
 	// a sysctl segment regex, concatenated with dots to form a sysctl name
 	SysctlSegmentFmt string = "[a-z0-9]([-_a-z0-9]*[a-z0-9])?"
 
-	// a sysctl name regex
-	SysctlFmt string = "(" + SysctlSegmentFmt + "\\.)*" + SysctlSegmentFmt
+	// a sysctl segment regex contains a slash, concatenated with dots to form a sysctl name
+	SysctlSegmentContainSlashFmt string = "[a-z0-9]([-_/a-z0-9]*[a-z0-9])?"
 
 	// the maximal length of a sysctl name
 	SysctlMaxLength int = 253
 )
 
-var sysctlRegexp = regexp.MustCompile("^" + SysctlFmt + "$")
-
 // IsValidSysctlName checks that the given string is a valid sysctl name,
 // i.e. matches SysctlFmt.
-func IsValidSysctlName(name string) bool {
+func IsValidSysctlName(name string, isContainSlash bool) bool {
 	if len(name) > SysctlMaxLength {
 		return false
 	}
+	sysctlFmt := getSysctlFmt(isContainSlash)
+	sysctlRegexp := regexp.MustCompile("^" + sysctlFmt + "$")
 	return sysctlRegexp.MatchString(name)
 }
 
-func validateSysctls(sysctls []core.Sysctl, fldPath *field.Path) field.ErrorList {
+func getSysctlFmt(isContainSlash bool) string {
```

Comment:
don't construct these dynamically, have two package var regexes and select between them

Reply:
Updated.